iroh-net/src/magicsock/endpoint.rs

```diff
@@ -151,6 +152,14 @@ impl Endpoint {
 
     /// Returns the address(es) that should be used for sending the next packet.
     /// Zero, one, or both of UDP address and DERP addr may be non-zero.
+    #[cfg(feature = "derp-only")]
+    fn addr_for_send(&mut self, _now: &Instant) -> (Option<SocketAddr>, Option<u16>, bool) {
```

Comment:
I got a slight preference for doing the feature flag distinction only inside the fn. Just need to do something like `let _ = now` to get rid of the unused warning.